packages/actions/src/helpers/storage.ts

```diff
@@ -251,8 +251,8 @@ export const openMultiPartUploadAPI = async (
   ceremonyId: number,
   token: string,
 ) => {
-  const url = new URL(`${process.env.API_URL}/storage/start-multipart-upload`);
-  url.search = new URLSearchParams({ ceremonyId: ceremonyId.toString() }).toString();
+  const url = new URL(`${process.env.API_URL}/storage/multipart/start`);
+  url.search = new URLSearchParams({ id: ceremonyId.toString() }).toString();
```

Comment:
Same thing here, we need to add `userId` in the URL search params. Remember to follow the same steps as the previous comment